internal/common/shared/iigopresidentreturn.go

```diff
@@ -0,0 +1,20 @@
+package shared
+
+// PresidentReturnContentType is the index of the return content that should be retrieved if the action was taken.
+type PresidentReturnContentType = int
+
+const (
+	PresidentAllocation PresidentReturnContentType = iota
+	PresidentTaxation
+	PresidentRuleProposal
+	PresidentSpeakerSalary
+)
+
+// PresidentReturnContent is a general datastructure used for president return type
+type PresidentReturnContent struct {
+	T             PresidentReturnContentType
```

Comment:
What does T stand for? Type?

Reply:
yeah. that was taken from the communications

Reply:
but I opened another pull request to outgoing instead of main, so maybe you could do the review there? #27